runtime/etc/Makefile.mli-static

```diff
@@ -21,7 +21,14 @@
 
 include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
-CHPL_MLI_INCS = -I$(CHPL_ZMQ_LOC)/include
+ifdef CHPL_ZMQ_HOME
+CHPL_MLI_INCS = -I$(CHPL_ZMQ_HOME)/include
+CHPL_ZMQ_LIB_HOME = $(CHPL_ZMQ_HOME)/lib
+CHPL_ZMQ_LIB = $(CHPL_ZMQ_LIB_HOME)/libzmq.a
+CHPL_ZMQ_DASH_L = -L$(CHPL_ZMQ_LIB_HOME)
```

Comment:
Now that I think about it, I would prefer "SEARCH_PATH", because one could easily conflate "-l" and "-L" when reading this.